community/server/src/docs/ops/security.asciidoc

```diff
@@ -129,8 +126,9 @@ This provides a number of advantages:
   Allow from 192.168.0
 </Proxy>
 ---------------
-
-While equivalent functionality can be implemented with Neo4j's +SecurityRule+ plugins (see above), for operations professionals configuring servers like Apache is often preferable to developing plugins. However it should be noted that where both approaches are used, they will work harmoniously providing the behavior is consistent across proxy server and +SecurityRule+ plugins.
++
+While equivalent functionality can be implemented with Neo4j's +SecurityRule+ plugins (see above), for operations professionals configuring servers like Apache is often preferable to developing plugins.
+However it should be noted that where both approaches are used, they will work harmoniously providing the behavior is consistent across proxy server and +SecurityRule+ plugins.
```

Comment:
However, it should be noted that in cases where both approaches are being used, they will work harmoniously, provided that the behavior is consistent across proxy server and +SecurityRule+ plugins.